searchcore/src/vespa/searchcore/proton/server/pendinglidtracker.cpp

```diff
@@ -38,7 +37,7 @@ PendingLidTracker::consume(uint32_t lid) {
 
 void
 PendingLidTracker::waitForConsumedLid(uint32_t lid) {
-    LockGuard guard(_mutex);
+    std::unique_lock<std::mutex> guard(_mutex);
```

Comment:
Why are you still using unique_lock here?

Reply:
Because we need a std::unique_lock when waiting on a std::condition_variable.